package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-config-edx",
+  "version": "0.0.1",
+  "description": "ESLint config for edX JavaScript code.",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint ."
```

Comment:
I actually really like that the lint repo passes it's own linter!  It would probably prevent errors from creeping in too.  For a while I had a linter running on my gulp config file and it caught some errors early on!